src/displayBackend/wayland/Display.cpp

```diff
@@ -530,7 +535,7 @@ void Display::dispatchThread()
 				DLOG(mLog, DEBUG) << "Dispatch events: " << val;
 			}
 
-			if (wl_display_flush(mWlDisplay) < 0)
+			if (wl_display_flush(mWlDisplay) < 0 && errno != EAGAIN)
```

Comment:
why not display->flush?

Reply:
Keep it as is in case we will remove display->flush